### DISCUSSION TITLE
Is it possible to convert a derived public key to DER format?

### DISCUSSION THREAD

**Discussion by koredefashokun** [paulmillr/noble-curves#134] [Q&A] [answered]
First, great work on this library!

I've generated a p256 keypair, and would like to retrieve the public key in the DER format. Is there an easy way to do this with this library. If not, what is the easiest way to achieve this result?

Thank you!

**Comment by paulmillr** [comment, marked as answer]
The default hex / bytes output are DER-encoded, I believe.

**Comment by koredefashokun** [comment]
You're right. The mistake I was making was using viem's `bytesToHex` instead of the one provided by the library. Thanks for the quick reply!
